Installers/Electron/src/utils.js

```diff
@@ -73,8 +73,8 @@ class Utils {
                 });
                 return path;
             },
-            "darwin": () => "/Applications/Discord.app",
-            "linux": () => "" // TODO
+            "darwin": () => "/Applications/Discord.app/Contents",
+            "linux": () => "/usr/share/discord-canary"
```

Comment:
All the paths here are to the stable release, so maybe this could be done for Linux here.
And then, add an option later to select between Stable, PTB or Canary for each platform could be a possibility to extend the support.